app/src/test/java/com/github/palFinderTeam/palfinder/profile/MockProfileService.kt

```diff
@@ -52,6 +52,31 @@ class MockProfileService : ProfileService {
             Response.Failure(e.message.orEmpty())
         }
     }
+
+    override suspend fun muteMeetup(user: ProfileUser, meetup: String): Response<Unit> {
+        return try {
+            if (!user.canMuteMeetup(meetup)) {
+                return Response.Failure("Cannot mute this meetup.")
+            }
+            db[user.uuid] = user.copy(mutedMeetups= user.following.plus(meetup))
+            Response.Success(Unit)
+        } catch (e: Exception) {
+            Response.Failure(e.message.orEmpty())
+        }
+    }
+
+    override suspend fun unMuteMeetup(user: ProfileUser, meetup: String): Response<Unit> {
+        return try {
+            if (!user.canUnMuteMeetup(meetup)) {
+                return Response.Failure("Cannot unfollow this user.")
```

Comment:
same problem, we should maybe make it more consistent with the real utility of this function ?